R/dbi-connection.R

```diff
@@ -220,7 +220,7 @@ setMethod("dbListFields", c("OdbcConnection", "character"),
 setMethod("dbGetInfo", "OdbcConnection",
   function(dbObj, ...) {
     info <- connection_info(dbObj@ptr)
-    structure(info, class = c(info$dbms.name, "driver_info", "list"))
```

Comment:
Since it's now just a "list", you can yeet the class attribute 😄 (and same below)

Reply:
So the `info$dbms.name` class is unneeded as well?

Reply:
Oh hmmm, that's a good question. But I don't think so since that's usually an S4 class and here we're making an S3 class.

Reply:
I double checked this by looking at the `NAMESPACE` to see what S3 generics we have methods for. As far as I can tell, the only S3 generic is `odbcDataType()`, and that's always called on the connection object.